lib/mastodon/github.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Mastodon
+  module Github
+    module_function
+
+    def host
+      'https://github.com'
+    end
+
+    def author
+      'tootsuite'
+    end
+
+    def repository
+      'mastodon'
+    end
+
+    def link_title
+      [author, repository].join('/')
+    end
+
+    def to_s
```

Comment:
Should be named `link_url` or something like that. There is no actual reason that the link title is not `to_s` and the URL is not `link_url`. Name them in a consistent rule then.